kubejs/server_scripts/base/entity/spawn/equipment.js

```diff
@@ -0,0 +1,64 @@
+EntityEvents.spawned((event) => {
```

Comment:
Perhaps rename the folder `spawn` -> `spawned` to match the event key?

Reply:
Oh good thinking